collections/_article/maintenance-release-godot-4-1-3.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Maintenance release: Godot 4.1.3"
+excerpt: "With stability in mind, Godot contributors offer yet another set of changes for Godot 4.1 users, addressing a variety of bugs related to rendering, input, GUI, and platform support."
+categories: ["release"]
+author: Yuri Sizov
+image: /storage/blog/covers/maintenance-release-godot-4-1-3.webp
+image_caption_title: Meowing Point
+image_caption_description: A game by yofrancisco
+date: 2023-11-02 11:00:00
+---
+
+After about a week of testing Godot 4.1.3 is ready for production use. As Godot 4.2 [is being finalized](/article/dev-snapshot-godot-4-2-beta-4) we stay committed to provide support for users remaining on the current stable version of the engine. Our architecture and approach to release management allow us to easily pick many bugfixes and smaller improvements from the upcoming release and apply them to previous releases without compromising their stability.
```

Comment:
```suggestion
After about a week of testing, Godot 4.1.3 is ready for production use. As Godot 4.2 [is being finalized](/article/dev-snapshot-godot-4-2-beta-4) we stay committed to providing support for users remaining on the current stable version of the engine. Our architecture and approach to release management allow us to easily pick many bugfixes and smaller improvements from the upcoming release and apply them to previous releases without compromising their stability.
```

Reply:
You know, that's what I wrote initially but then thought it didn't sound right.

Reply:
The comma or the "providing"?

Reply:
The "providing".

Reply:
I *think* "providing" is correct. I'm considering other phrases, like if someone takes over a company, they might say "we're committed to running the company as before", not "we're committed to run the company as before".

Reply:
Well your suggestion matches my gut feeling, apparently. So I'll update it :)